dependencies/buster/redfish_client/patches/decrease-excon-dependency.patch

```diff
@@ -0,0 +1,12 @@
+Pick any version of HTTP library.
+--- a/redfish_client.gemspec
++++ b/redfish_client.gemspec
+@@ -29,7 +29,7 @@
+ 
+   spec.required_ruby_version = "~> 2.1"
+ 
+-  spec.add_runtime_dependency "excon", "~> 0.71"
++  spec.add_runtime_dependency "excon"
```

Comment:
why?

Reply:
This gem is in Debian in old version, it's a HTTP client library very unlikely it really needs the most recent one.

Reply:
Ah, okay, we don't build it ourself. Well, then.